README.md

```diff
@@ -109,17 +109,45 @@ You can also cross-compile if necessary:
 GOOS=windows GOARCH=amd64 make build
 ```
 
+#### Unit tests
+
 In order to run the Unit Tests for the provider, you can run:
 
 ```sh
 $ make test
 ```
 
-The majority of tests in the provider are Acceptance Tests - which provisions real resources in Azure. It's possible to run the entire acceptance test suite by running `make testacc` - however it's likely you'll want to run a subset, which you can do using a prefix, by running:
+With VSCode Golang extension you can also run and debug the tests using `run test`, `debug test` `run package tests`, `run file tests` buttons.
 
-```sh
-make testacc SERVICE='resource' TESTARGS='-run=TestAccAzureRMResourceGroup' TESTTIMEOUT='60m'
-```
+#### Acceptance tests
+
+The majority of tests in the provider are acceptance tests - which provisions real resources in Azure Devops and Azure. To run any acceptance tests you need to set `AZDO_ORG_SERVICE_URL`, `AZDO_PERSONAL_ACCESS_TOKEN` environment variables, some test have additional environment variables required to run. You can find out the required environment variables by running the test.
+
+The several options to run the tests are:
+
+* Run the entire acceptance test suite
+
+  ```sh
+  make testacc
+  ```
+
+* Run a subset using a prefix
+
+  ```sh
+  make testacc TESTARGS='-run=TestAccBuildDefinitionBitbucket_Create'
+  ```
+
```

Comment:
Perhaps we should introduce a variable in the makefile so that people can limit the executed acceptance tests via tags like so:

`make testacc TESTTAGS='core data_agent_pool exclude_resource_group'`

You can achieve this already by executing acceptance tests with the `acctests.ps1` script using the `-Tags` parameter but I think because of consistency reasons we should support this in the makefile as well.

Reply:
Like that?

Reply:
Of course! This is great! 👍 🚀